webapp/src/components/screen_source_modal/component.tsx

```diff
@@ -133,7 +134,17 @@ export default class ScreenSourceModal extends React.PureComponent<Props, State>
                             src={source.thumbnailURL}
                         />
                     </div>
-                    <span style={this.style.sourceLabel as CSSProperties}>{source.name}</span>
+
+                    <OverlayTrigger
```

Comment:
If I'm not mistaken, I think new code should use the new floating-ui framework? (There was a PR by Daniel adding a new tooltip we should use, I think...)
I'll leave it up to you though, considering this is just a small addition.

Reply:
I think these should be migrated to WithTooltip (https://mattermost.atlassian.net/browse/MM-57219). Something for next Fixathon I suppose.